RectangleStamped[] map_rects
RectangleStamped[] unknown_rects
